ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -44,13 +44,13 @@ type LinuxParams struct {
 // See https://github.com/aws/amazon-ecs-cli/issues/797
 const SecretLabelPrefix = "ecs-local.secret"
 
+// TaskDefinitionLabelType represents the type of option used to
```

Comment:
Usually for constant comments we want to either:
* Give a single comment to an entire block of constants (like [here](https://github.com/golang/go/blob/master/src/text/scanner/scanner.go#L74-L88))
* Or individually give comments to each constant (like [here](https://github.com/golang/go/blob/master/src/unicode/tables.go#L5-L6))

so for you it's:
```go
const (
     // TaskDefinitionLabelType represents the type of option used to
     // transform a task definition to a compose file e.g. remoteFile, localFile.
     TaskDefinitionLabelType  = "ecs-local.task.type"

     // TaskDefinitionLabelValue represents the value of the option e.g. file path, arn, family.
     TaskDefinitionLabelValue = "ecs-local.task.value"
)
```

Reply:
I see. Thanks for the advice. Done